snafu/benchmarks/uperf/uperf.py

```diff
@@ -273,21 +273,21 @@ def get_results_from_stdout(stdout: UperfStdout) -> List[UperfStat]:
             timestamp, num_bytes, ops = result.timestamp, result.bytes, result.ops
 
             norm_ops = ops - prev_ops
-            if norm_ops == 0:
-                norm_ltcy = 0.0
+            if norm_ops == 0 or prev_timestamp == 0.0:
+                pass
```

Comment:
LGTM, perhaps instead of the "pass" use "if norm_ops != 0 and prev_timestamp != 0.0:"

Reply:
makes sense, thanks